src/main/java/frc/robot/subsystems/swerve/SwerveModule.java

```diff
@@ -88,8 +88,8 @@ private static double placeInAppropriate0To360Scope(
         return newAngle;
     }
 
-    public int[] getEncoderVelocities() {
-        return new int[] {
+    public double[] getEncoderVelocities() {
+        return new double[] {
```

Comment:
CTRE updated thier code? this compiles and works through simulation?

Reply:
yep